<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Numerical_Methods</class>
 <widget class="QMainWindow" name="Numerical_Methods">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Numerical_Methods</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="met11">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>250</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>MÉTODO RUNGE-KUTTA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>                               NUMERICAL METHODS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="met1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>ITEERACIÓN DE PUNTO FIJO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="met2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>MÉTODO DE NEWTON-RAPHSON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="met10">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>150</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>MÉTODO DE EULER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="met5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>INTERPOLACIÓN DE LAGRANGE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="met7">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>REGRESIÓN LINEAL MULTIPLE </string>
    </property>
   </widget>
   <widget class="QPushButton" name="met6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>MÉTODO DE REGRESIÓN LINEAL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="met9">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>INTEGRACIÓN NUMÉRICA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="met4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>MÉTODO GAUSS-SEIDEL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="met8">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>DIFERENCIACIÓN NUMÉRICA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="salir">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>350</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>SALIR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="met3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>ELIMINACIÓN GAUSSIANA</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
